templates/strapi/files/.platform.app.yaml

```diff
@@ -0,0 +1,45 @@
+# This file describes an application. You can have multiple applications
```

Comment:
There's no cron section in here, but Strapi's configuration files talk about cron.  Does it have its own internal cron logic that ignores system cron?  (Seriously people???)

If so, that may be worth documenting, um, somewhere.  Either here or the README, not sure where.